bindings/nodejs/tests/suites/async.suite.mjs

```diff
@@ -19,15 +19,31 @@
 
 import { randomUUID } from 'node:crypto'
 import { test } from 'vitest'
+import { generateBytes } from '../utils.mjs'
 
-export function run(operator) {
+export function run(op) {
   test('async stat not exist files', async () => {
     const filename = `random_file_${randomUUID()}`
 
     try {
-      await operator.stat(filename)
+      await op.stat(filename)
     } catch (error) {
       assert.include(error.message, 'NotFound')
     }
   })
+
+  test('async writer', async () => {
```

Comment:
This test will fail on most storage even when they support `writeCanMulti` since services may have it's own `write_min_size`. Rust core implement those tests by adding a `buffer`.

Reply:
All other Storage services are fine. s3 keeps reporting that the size is too small on close. 
I have changed the size to 5MB and it still reports the same error.